include/ozo/query_builder.h

```diff
@@ -179,10 +179,14 @@ constexpr auto operator +(query_element<LhsValueT, LhsTagT>&& lhs, RhsValueT&& r
 
 namespace literals {
 
+#ifdef __GNUG__
+
```

Comment:
Sorry, but this is wrong way. No source code should be modified on this PR. This PR should modify CMake files only. Enabling or disabling the library’s features is a completely different topic.

Reply:
Well, the idea is of course to disable it for compilers that don't support this non-standard option. It won't change anything for supported compilers.

I've removed the guard, though I do think it should be added again because this prohibits people from using the library using different compilers.